app/pages/SplitPaymentPage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object SplitPaymentPage extends QuestionPage[Boolean] {
+
+  def path: JsPath = JsPath \ toString
+
+  override def toString: String = "split-payment"
+
```

Comment:
If the user selects "No" it might be useful to blank the "Additional DAN" fields in case the user has come back to this screen after selecting to split earlier.